test/unit/filtered-list.test.ts

```diff
@@ -0,0 +1,64 @@
+import { expect, fixture, html } from '@open-wc/testing';
+import { Filterlist } from '../../src/filtered-list.js';
+
+describe('filtered-list', () => {
+  let element: Filterlist;
+  const listItems = [
+    { prim: 'item1', sec: 'item1sec' },
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/openscd/open-scd/pullRequest?prid=7203429)
```suggestion
    { prim: "item1", sec: 'item1sec' },
```